### ISSUE TITLE
Use standard package directly for linting

### ISSUE THREAD

**Issue by Cruikshanks** [DEFRA/sroc-service-team#6]
Currently, the charging module API is using eslint for linting of the code. It appears to be using the StandardJS ruleset.

When you run `npm run lint` no issues are found. If you run `standard` against the repo though there are issues.

[StandardJS](https://github.com/DEFRA/software-development-standards/blob/master/standards/javascript_standards.md) is our standard at Defra and we really should be using it and there is no real need to go via eslint to do so.

For reference, the Tactical Charging Service is using standard directly if we need an example.

**Comment by Cruikshanks** [comment]
I did have a go at this. I deleted all the eslint config files and all the eslint dev dependencies. First issue I got was this

```bash
Failed to load plugin 'import' declared in '--config » eslint-config-standard': Cannot find module 'eslint'
```

It is reported as an [issue on the Standard repo](https://github.com/standard/standard/issues/1439) and the say the solution if to delete your node_modules and package-lock.json and run `npm install` again.

That worked, `npm run lint` completed successfully. But after doing this some of the tests started failing (something about 'window' not being found or undefined. Apologies, I no longer have the error to hand).

So my first attempt failed utterly to I reverted everything and logged this issue!
